python/setup.py.in

```diff
@@ -24,7 +24,7 @@ if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
 # the prefix is sys.prefix which should always be usr
-paddle_bin_dir = 'local/opt/paddle/bin'
+paddle_bin_dir = 'opt/paddle/bin'
```

Comment:
I think `make install` and `pip install` will install to different default location doesn't harm using.
1. Users could either use `make install` or `pip install` to install and then use paddle after that.
2. The default install path is determined by cmake and pip, we shouldn't interfere with it. Let cmake and pip do their work.
3. Maybe add some documentation about how to use `--prefix` parameter when installing, and the default installation behaviour?